src/components/Profile/ProfileLink/ProfileLink.tsx

```diff
@@ -29,8 +29,10 @@ const ProfileLink = ({ type, link }: ProfileLinkProps) => {
 
   return (
     <div className="profile__header__link">
-      {icon}
-      <a href={`https://${link}`}>{type}</a>
+      <a href={`https://${link}`} target="_blank" rel="noopener noreferrer">
```

Comment:
Nit pick: Please mention and explain this type of changes in the PR. While this is more than welcome, it deserves a mention for reviewers and for posterity.